lib/manageiq/api/common/request.rb

```diff
@@ -47,7 +47,21 @@ def user
         end
 
         def to_h
-          {:headers => headers.to_h, :original_url => original_url}
+          {:headers => forwardable, :original_url => original_url}
+        end
+
+        def forwardable
```

Comment:
Should this be private?

Reply:
It can't be because we call it up in the self.current_forwardable method